backend/models/users.js

```diff
@@ -6,7 +6,10 @@ const users = {
    * @returns all users
    */
   findAll: async () => {
-    const queryString = `SELECT * FROM users`;
+    const queryString = `
+      SELECT id, first_name AS firstname, last_name AS lastname, email,
+      postal_code AS postalcode, city, country, phone, premium
+      FROM users`;
     const [rows] = await promisePool.query(queryString);
     return rows === undefined ? null : rows;
```

Comment:
This returns an array (which is never undefined) so could simplify
```suggestion
    return rows;
```